resources/css/header.css

```diff
@@ -72,6 +72,7 @@ HEADER STYLES
   width: var(--toc-width);
   padding: 0 var(--space-x) 0 10px;
   border-left: 1px solid var(--brand-mute-color);
+  margin: auto;
```

Comment:
```suggestion
  margin-block: auto;
```

Reply:
https://caniuse.com/?search=margin-block
Are you sure? This property seems to not be supported at all in IE, I think it won't concern any of our users, but what is the advantage of margin-block over simple margin in this case ( I'm going to read more about it on my side ) ?
Sorry to nitpick but I just want to understand 😅

Reply:
Well anyway, I guess a lot of other things are broken on the site under IE. So let's gooo !